action/protocol/execution/evm/evmstatedbadapter.go

```diff
@@ -428,29 +438,36 @@ func (stateDB *StateDBAdapter) AddPreimage(hash common.Hash, preimage []byte) {
 }
 
 // ForEachStorage loops each storage
-func (stateDB *StateDBAdapter) ForEachStorage(addr common.Address, cb func(common.Hash, common.Hash) bool) {
-	ctt, err := stateDB.Contract(hash.BytesToHash160(addr[:]))
+func (stateDB *StateDBAdapter) ForEachStorage(addr common.Address, cb func(common.Hash, common.Hash) bool) error {
+	ctt, err := stateDB.getContract(hash.BytesToHash160(addr[:]))
```

Comment:
refer to
https://github.com/iotexproject/go-ethereum/commit/36f81118f6d00713dbf8d1caf168ad937e0f1a90